cmd/storagenode/main.go

```diff
@@ -387,7 +399,7 @@ func dashCmd(cmd *cobra.Command, args []string) (err error) {
 
 		fmt.Fprintf(color.Output, "Node ID: %s\n", color.YellowString(data.GetNodeId()))
 
-		if data.GetConnection() {
+		if online {
```

Comment:
Could we call this on a timer instead, in case of a temporary outage?

Reply:
I'm gonna put it on a ticker 👍